sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 4.0.0-preview.7 (2019-10-07)
+- Added @azure/eslint-plugin-azure-sdk.
+- Added a "Deploy to Azure" button to the README.
+- Updated README examples to latest API changes.
```

Comment:
None of the above 3 points tell a customer why they should update to preview 7. Internal improvements like these need not be called out in the changelog.

@jonathandturner, @sadasant  thoughts on if we should have content like this? My worry is that the actual things that matter to the customer gets buried inside these.

If we remove this then I would make `API Changes` as the main bullet point and have the api changes as sub bullet points

Reply:
That's fine too. I think people tend to scan through the changelog looking to see if something will impact them.  I agree that having the API changes first helps them do that, and we can other bits last.